src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -65,8 +65,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String userEmail = userService.getCurrentUser().getEmail();
     String aboutMe = Jsoup.clean(request.getParameter("about-me"), Whitelist.none());
-
-    User user = new User(userEmail, aboutMe);
+    
+    User user = new User(userEmail, aboutMe, password, firstName, lastName, imgUrl);
```

Comment:
You can get the firstName, lastName and imgUrl like so:
```suggestion
    String firstName = request.getParameter("firstName");
    String lastName = request.getParameter("lastName");
    String imgUrl = request.getParameter("imgUrl");
    User user = new User(userEmail, aboutMe, null, firstName, lastName, imgUrl);
```

You can also omit the password here.